certification/internal/bindings/helpers.go

```diff
@@ -0,0 +1,162 @@
+package bindings
+
+import (
+	"fmt"
+	"os"
+	"reflect"
+	"strings"
+	"time"
+
+	"github.com/containers/podman/v3/libpod/define"
+	"github.com/containers/podman/v3/pkg/domain/entities"
+	"github.com/containers/podman/v3/pkg/inspect"
+	"github.com/docker/docker/api/types"
+	"github.com/opencontainers/go-digest"
+	specv1 "github.com/opencontainers/image-spec/specs-go/v1"
+)
+
+// getTarballName takes the name or ID of an image and
+// returns the name of an archive
+func getTarballName(nameOrID string) string {
+	var imageURL string
+	if strings.Contains(nameOrID, "@") {
+		imageURL = strings.Split(nameOrID, "@")[0]
+	}
+
+	if strings.Contains(nameOrID, ":") {
+		imageURL = strings.Split(nameOrID, ":")[0]
+	}
+
+	if imageURL == "" {
+		imageURL = nameOrID
+	}
+
+	if strings.Contains(imageURL, "/") {
+		imageName := strings.Split(imageURL, "/")
+		return fmt.Sprintf("%s.tar.gz", imageName[len(imageName)-1])
+	}
+
+	return fmt.Sprintf("%s.tar.gz", imageURL)
+}
+
+// The image pull path defaults to /tmp
+// when the IMAGE_PULL_PATH is not set
+func imagePullPath() string {
+	pullPath := "/tmp"
+	overridePath, ok := os.LookupEnv("IMAGE_PULL_PATH")
+	if ok && overridePath != "" {
+		pullPath = overridePath
+	}
+	return pullPath
+}
+
+func parseTime(stringTime string) *time.Time {
+	t, err := time.Parse(time.RFC3339, stringTime)
+	if err != nil {
+		return nil
+	}
+	return &t
+}
+
+func buildInspectImageReport(imageData interface{}) *InspectImageReport {
```

Comment:
Add a comment describing the behavior

Reply:
The reason I ask for this is because the input and output types appear similar in their description (the parameter name, and the return type), so the comment would help clarify to us the exact reason for this function existing without having to understand the code's behavior. This also applies to the next location.

Reply:
this function has been removed.